tests/Integration/APITest.php

```diff
@@ -1285,7 +1294,7 @@ protected function setAdminUser()
     public function provideContainerConfig()
     {
         return array(
-            'Piwik\Access' => new FakeAccess()
+            'Piwik\Access' => new FakeAccessTagManager()
```

Comment:
@snake14 We were already using a `FakeAccess` from core but that function didn't check if you have `tagmanager_use_custom_templates` capability or not instead returns null if you have write access, so I extended the core mock class to check for `tagmanager_use_custom_templates` capability if requested